tools/cpp/osx_cc_configure.bzl

```diff
@@ -140,7 +139,6 @@ def configure_osx_toolchain(repository_ctx, overriden_tools):
                  error_msg)
 
         escaped_include_paths = _get_escaped_xcode_cxx_inc_directories(repository_ctx, cc, xcode_toolchains)
-        write_builtin_include_directory_paths(repository_ctx, cc, escaped_include_paths)
```

Comment:
I'm not sure if it's safe to remove the include directories altogether (see https://github.com/bazelbuild/bazel/issues/10187#issuecomment-556205226).

Would it be sufficient to just replace the `cc` path here with a fixed string? e.g.

```
write_builtin_include_directory_paths(repository_ctx, "wrapped_clang", escaped_include_paths)
```

Reply:
What from that comment specifically do you mean makes it unsafe?

I don't think replacing `cc` with `wrapped_clang` would affect anything here since the core piece I want to remove is the explicit path to Xcode.app in the generated file:

```
#include <...> search starts here:
 /usr/local/include
 /Applications/Xcode-11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1
 /Applications/Xcode-11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/11.0.0/include
 /Applications/Xcode-11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode-11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include
 /Applications/Xcode-11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks (framework directory)
```

Here I would expect the output to be the same whether you can `clang` or `wrapped_clang` on macOS